doc/source/parameters/dem/insertion_info.rst

```diff
@@ -15,6 +15,8 @@ In this subsection, insertion methods which are ``volume``, ``plane``, ``list``
 
     # Every method
     set insertion frequency                            = 20000
+    set clearing particles                             = false
```

Comment:
I'm not sure I like the name clearing. The deal.II nomenclature is "remove" for particles. I would rather we remain coherent.